src/sdk.js

```diff
@@ -5,7 +5,7 @@
 class Smartcar {
   /**
    * @callback OnComplete
-   * @param {?Error} error - something went wrong in the authorization flow; this
+   * @param {Error} [error] - something went wrong in Connect; this
```

Comment:
this field is not optional, it's just nullable 
```suggestion
   * @param {?Error} error - something went wrong in Connect; this
```

Reply:
:man_facepalming: thanks! Didn't know that.

Reply:
Fixed